app/controllers/application_controller.rb

```diff
@@ -5,6 +5,182 @@ class ApplicationController < Sinatra::Base
   configure do
     set :public_folder, 'public'
     set :views, 'app/views'
+    enable :sessions
+    set :session_secret, "Fwitter is secure"
   end
 
-end
\ No newline at end of file
+  get '/' do
+    erb :index
+  end
+
+  get '/tweets' do
+# <<<<<<< HEAD
+
+
+# =======
+    if is_logged_in?
+      @user = current_user
+      @tweets = Tweet.all
+      erb :'tweets/tweets'
+    else
+      redirect to '/login'
+    end
+# >>>>>>> f9e125e886b061a6dbed2c622dc8cd06e3ae2757
+  end
+
+  get '/tweets/new' do
+    if is_logged_in?
+      erb :'tweets/create_tweet'
+    else
+      redirect to '/login'
+    end
+  end
+
+  get '/tweets/:id' do
+    if is_logged_in?
+      @tweet = Tweet.find_by_id(params[:id])
+      erb :'/tweets/show_tweet'
+    else
+      redirect to '/login'
+    end
+
+  end
+
+  get '/tweets/:id/edit' do
+    if is_logged_in?
+      @tweet = Tweet.find_by_id(params[:id])
+
+      if @tweet.user_id == current_user.id
+        erb :'tweets/edit_tweet'
+      else
+        redirect to '/tweets'
+      end
+    else
+      redirect to '/login'
+    end
+  end
+
+  post '/tweets' do
+    if params[:content] == nil || params[:content] == ""
+      redirect to '/tweets/new'
+    else
+      if is_logged_in?
+        @tweet = current_user.tweets.create(content: params[:content])
+        redirect("/tweets/#{@tweet.id}")
+      else
+        redirect to '/login'
+      end
+    end
+  end
+
+
+
+  patch '/tweets/:id' do
+    if params[:content] == "" || params[:content] == nil
+      redirect to "/tweets/#{params[:id]}/edit"
+    else
+      @tweet = Tweet.find_by_id(params[:id])
+      @tweet.content = params[:content]
```

Comment:
`@tweet.update(content: params[:content])` would work here instead of setting the value and then saving